operator/src/main/java/oracle/kubernetes/weblogic/domain/model/Domain.java

```diff
@@ -538,6 +538,15 @@ public String getWdtConfigMap() {
         .map(Configuration::getSecrets).orElse(spec.getConfigOverrideSecrets());
   }
 
+  /**
+   * Returns the model home directory of the domain.
+   *
+   * @return model home directory
+   */
+  public String getModelHome() {
+    return Optional.ofNullable(spec.getConfiguration())
+        .map(Configuration::getModel).map(Model::getModelHome).orElse(null);
```

Comment:
This needs to return the default value when the value is unset.

Reply:
Oh, I see you process the default in the script... It would be clearer to me if it was here similarly to other values.